experimental/origin-graphql/src/mutations/web3/signMessage.js

```diff
@@ -0,0 +1,6 @@
+import contracts from '../../contracts'
+
+export default async (_, { address, message }) => {
+  //TODO: get this right
```

Comment:
is there an issue with this?

Reply:
It just worked without Metamask prompting user to sign (I was expecting a popup window). Was planning to get to the bottom of this in the next PR.